components/components/view-modes/highlighted-small/_node-highlighted-small.css

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file
+ * This file is used to style the 'highlighted-small' view mode.
+ */
+
+/*
+  Highlighted Small
+
+  .node--view-mode-highlighted-small - This presents the 'Highlighted Small' view mode.
+
+  Markup: node--highlighted-small.html.twig
+
+  Style guide: components.view_modes.highlighted_small
+*/
+
+.node--view-mode-highlighted-small {
+  border: 1px solid #fcece7;
+}
+
+.node--view-mode-highlighted-small .node__meta {
+  padding: 1.5rem;
+}
+
+.node--view-mode-highlighted-small .node__meta > h2 {
+  margin-bottom: 1rem;
+}
+
+.node--view-mode-highlighted-small .node__meta > h2 .field--name-title {
```

Comment:
I'd vote for using shorter selectors like .node--view-mode-highlighted-small .field--name-title (if possible).

Reply:
+1 to shorter selectors

Reply:
That shorter selector worked. Thanks.